### PR TITLE
MAT-4516 return type validation for population definitions

### PR DESCRIPTION
**Pull request by adongare**
## MADiE PR

Jira Ticket: [MAT-4516](https://jira.cms.gov/browse/MAT-4516)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

Screen Shot 2022-08-24 at 12 20 51 PM

### CODE REVIEW

File: src/validations/MeasureGroupSchemaValidator.ts [hunk 1]
```diff
@@ -1,121 +1,132 @@
...
-                .oneOf(Object.values(AggregateFunctionType))
-                .required(
-                  "Aggregate Method is required for a measure observation"
+        }
+        return true;
+      },
+    };
+  };
+
+  const createPopulationSchema = (
```
Comment (sb-cecilialiu) [inline src/validations/MeasureGroupSchemaValidator.ts]: Nice!

**Review by sb-cecilialiu — DISMISSED** [review]

File: src/validations/MeasureGroupSchemaValidator.ts (hunk 1 above)
Comment (adongare) [reply src/validations/MeasureGroupSchemaValidator.ts]: thank you!

**Comment by jkotanchik-SB** [comment]
Seeing the error message live, I find it confusing. Which "Encounter" should I align with? 
@adongare Does that message live here or in the backend service?

**Comment by adongare** [comment]
@jkotanchik-SB It doesn't need to align with a specific encounter. Just return type should be of type `Encounter` resource. But yeah, that message doesn't make much sense(copied from the story).

update: got the new message from ui/ux

**Review by RohitKandimalla — DISMISSED** [review]

**Review by jkotanchik-SB — APPROVED** [review]

**Pull request merged**